### PR TITLE
Wish List and Product Added

### PR DESCRIPTION
**Pull request by ashifkhn**
Ecom Website WishList and Cart Added. Please Review:

Inspect the deploy log: https://app.netlify.com/sites/d-gadget-store/deploys/620ff226c1b07f00080adfe4
Browse the preview: https://deploy-preview-5--d-gadget-store.netlify.app/

### CODE REVIEW

File: component/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+.heading {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+nav {
+  display: flex;
```
Comment (raieegulhane) [inline component/cart/cart.css:28]: personal suggestion: you can make a class for flex properties and use it wherever needed

File: component/cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/b78dcc4774.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
```
Comment (raieegulhane) [inline component/cart/cart.html:13]: have you not used your component lib for ecomm? i cannot see it linked here

File: css/button.css [hunk 3]
```diff
@@ -1,26 +1,34 @@
...
 }
 
 .btnRed {
-	background-color: rgb(211, 52, 52);
+  background-color: rgb(211, 52, 52);
 }
 
 .btnGreen {
-	background-color: rgb(53, 179, 128);
+  background-color: rgb(53, 179, 128);
```
Comment (raieegulhane) [inline css/button.css:21]: variables for color is preferred

**Review by raieegulhane — COMMENTED** [review]
please check the comments. rest looks good

File: component/cart/cart.css (hunk 1 above)
Comment (ashifkhn) [reply component/cart/cart.css:28]: I will. Thanks

**Review by Pratham82 — COMMENTED** [review]
1.  Add padding to the like icon in wishlist component
2. Increase the size of clear button in product listing page
3. Convert rating component from checkboxes to radio buttons
Rest all looks good

**Pull request merged**